packages/engine/Source/Scene/PostProcessStage.js

```diff
@@ -94,9 +94,16 @@ import PostProcessStageSampleMode from "./PostProcessStageSampleMode.js";
 function PostProcessStage(options) {
   options = defaultValue(options, defaultValue.EMPTY_OBJECT);
   const {
+    name = createGuid(),
     fragmentShader,
+    uniforms,
     textureScale = 1.0,
+    forcePowerOfTwo = false,
+    sampleMode = PostProcessStageSampleMode.NEAREST,
```

Comment:
This method of setting default values is _not_ equivalent to the previous `defaultValue` calls when the value is `null`. I just want to make sure that's intended
![2024-11-27_16-12](https://github.com/user-attachments/assets/60637fd8-93e5-4646-895f-0d13431028a9)